app/routes/files.js

```diff
@@ -0,0 +1,118 @@
+const { getOrganisation } = require('../data')
+const { uploadFile } = require('../files')
+const { handleUploadError } = require('../files')
+const { parseUploadResponse } = require('../files')
+const { deleteFile } = require('../files')
+
+let sessionMetadata = []
```

Comment:
For purposes of spike on the stub this can stay but would need reworking for a real service

Reply:
Agree, this is exactly how this method ended up here. For real service we going to be using sessionMetadata on our servers I guess

Reply:
Frontend so will leave it up to consuming service 😁 

If we have to solve it for SFD then we can explore some options